src/react/account/AccountUserList.jsx

```diff
@@ -126,23 +172,43 @@ const AccountUserList = () => {
 
   return (
     <div style={{ height: '100%' }}>
-      <Table
-        columns={columns}
-        data={iamUsers}
-        defaultSortingKey={'userName'}
-        onBottom={() => {
-          if (hasNextPage && !isFetchingNextPage) {
-            return fetchNextPage();
-          }
-        }}
-      >
+      <Table columns={columns} data={iamUsers} defaultSortingKey={'userName'}>
         <TableHeader>
-          <Table.SearchWithQueryParams
-            displayedName={{
-              singular: 'user',
-              plural: 'users',
-            }}
```

Comment:
Maybe we should keep this in order to display `Total {iamUsers.length} user(s)`?